docs/administrator-documentation/moderne-dx/how-to-guides/configure-dx-organizations.md

```diff
@@ -76,20 +38,17 @@ The following table contains all of the variables/arguments you need to add to y
 
 | Variable Name                                     | Required | Default | Description                                                                                                                                                                                                                                      |
 |---------------------------------------------------|----------|---------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `MODERNE_DX_ORGANIZATION_URL`                     | `true`   |         | The URL of your GraphQL service that provides organization information. Cannot be combined with `MODERNE_DX_ORGANIZATION_REPOSCSV`.                                                                                                              |
-| `MODERNE_DX_ORGANIZATION_REPOSCSV`                | `true`   |         | The path of your `repos.csv` file that provides organization information. Cannot be combined with `MODERNE_DX_ORGANIZATION_URL`.                                                                                                                   |
+| `MODERNE_DX_ORGANIZATION_REPOSCSV`                | `true`   |         | The path of your `repos.csv` file that provides organization information. This could also be an unauthenticated HTTP/S URI in the form of `https://your-serve/repos.csv`.            |
+| `MODERNE_DX_ORGANIZATION_URL`                     | `false`   |         | The URL of your GraphQL service.                                                                                                              |
```

Comment:
This is going to be confusing for users. Since you said that the `reposCsv` can be an unauthenticated HTTP/S URI, they're going to be confused as to what this URL is for. 

Maybe move this to the bottom, or change it in such a way:

```suggestion
| `MODERNE_DX_ORGANIZATION_URL`                     | `false`   |         | The URL of an optional additional customization service. See [Additional customization](#additional-customization) for more information.                                                                                                              |
```

Also don't forget about `MODERNE_DX_ORGANIZATION_SKIPSSL`.